web/src/components/SubmissionsTable.vue

```diff
@@ -0,0 +1,139 @@
+<template>
+  <v-data-table
+    class="submissions-table row-pointer"
+    :headers="headers"
+    :items="items"
+    :search="searchValue"
+    sort-by="similarity"
+    sort-desc
+    hide-default-footer
+    disable-pagination
+    must-sort
+    fixed-header
+    @click:row="rowClicked"
+  >
+    <template #item.name="{ item }">
+      <div class="submission-info">
+        <div class="submission-name">
+          {{ item.name }}
+        </div>
+
+        <div class="submission-path text--secondary">
+          <v-icon small>mdi-file-document-outline</v-icon>

```

Comment:
I would only show the file path if the name is different from the (short)path.
We could also consider not showing the file path itself at all (since we try to guess the name based on the path anyway), or hiding it behind a tool tip, showing the full path.

Reply:
I would be consistent. If we don't show the file path it should be left out always.
I will add a tooltip showing the full path.

![image](https://user-images.githubusercontent.com/6013068/186379274-5ea990a6-7d26-4e79-99c2-de8ec9fb4dd3.png)